src/main/java/net/pms/formats/v2/AudioProperties.java

```diff
@@ -38,78 +39,42 @@
 	private static final Pattern intPattern = Pattern.compile("([\\+-]?\\d+)([eE][\\+-]?\\d+)?");
 //	private static final Pattern floatPattern = Pattern.compile("([\\+-]?\\d(\\.\\d*)?|\\.\\d+)([eE][\\+-]?(\\d(\\.\\d*)?|\\.\\d+))?");
 
-	public static final int BITRATE_DEFAULT = 8000;
-	public static final int NUMBEROFCHANNELS_DEFAULT = 2;
-	public static final int BITSPERSAMPLE_DEFAULT = 16;
-	public static final int AUDIODELAY_DEFAULT = 0;
-	public static final int SAMPLEFREQUENCY_DEFAULT = 48000;
-
-	private int bitRate;
 	private int numberOfChannels;
-	private int bitsperSample;
 	private int audioDelay;
 	private int sampleFrequency;
 
 	public int getAttribute(AudioAttribute attribute) {
 		switch (attribute) {
-			case BITRATE:
```

Comment:
I guess that you're right, it's up to @SubJunk to see if he consider it as possible to do for the next year, as i think that by now only 3 plugins are still working, at least partially.